ibis/omniscidb/tests/test_client.py

```diff
@@ -104,6 +104,74 @@ def test_union_op(alltypes):
         expr.compile()
 
 
+@pytest.mark.parametrize(
+    'cols_with_types',
+    [
+        {},
+        {'c': 'float64'},
+        {'c': 'float64', 'd': 'string', 'e': 'point', 'f': 'polygon'},
+    ],
+)
+def test_add_column(con, cols_with_types):
```

Comment:
create a new fixture that takes a con, and creates a table with a fixed schema, yields, then cleans up the table.